charts/mysqldump/templates/configmap-script.yaml

```diff
@@ -85,6 +85,13 @@ data:
     rcu=$?
     {{ end }}
 
+    {{ if .Values.upload.s3.enabled -}}
+    echo "upload files to s3 storage bucket {{ .Values.upload.s3.bucketname }}"
+    mc alias set s3 ${S3_ENDPOINT} ${S3_ACCESSKEY} ${S3_SECRET_KEY}
```

Comment:
please use "aws s3 cp" comnmand.
i'll add it to the container.

Reply:
completed